restli-tools/src/main/java/com/linkedin/restli/tools/idlcheck/CompatibilityInfo.java

```diff
@@ -65,7 +65,9 @@
     PAGING_ADDED(Level.COMPATIBLE, "Method added paging support"),
     PAGING_REMOVED(Level.INCOMPATIBLE, "Method removed paging support"),
     SERVICE_ERROR_ADDED(Level.INCOMPATIBLE, "Service error \"%s\" now applies"),
-    SERVICE_ERROR_REMOVED(Level.COMPATIBLE, "Service error \"%s\" no longer applies");
+    SERVICE_ERROR_REMOVED(Level.COMPATIBLE, "Service error \"%s\" no longer applies"),
+    BREAK_OLD_CLIENTS(Level.INCOMPATIBLE, "Deleting an schema is incompatible change, it breaks old clients"),
+    ANNOTATION_INCOMPATIBLE_CHANGE(Level.INCOMPATIBLE, "%s");
```

Comment:
Just a thought: Unlike other message, Annotation incompatible  message doesn't have a message handler and description, which actually made it more difficult to be code-searched

Reply:
This one is used for customer to define their custom compatibility rules, all the message info will be put here to replace "%s". Do you think we should provide something like - " Annotation incompatible change: "? Or maybe leave it as-is.

Reply:
I will agree with what you now proposed

Reply:
Updated.